pubspec.yaml

```diff
@@ -35,13 +35,13 @@ dependencies:
              # https://github.com/dart-lang/i18n/issues/759#issuecomment-1864316701
              # https://github.com/flutter/flutter/issues/117163
 
-  app_settings: ^5.0.0
+  app_settings: ^6.1.1
   collection: ^1.17.2
   convert: ^3.1.1
   crypto: ^3.0.3
   device_info_plus: ^11.2.0
   drift: ^2.23.0
-  file_picker: ^9.0.2
+  file_picker: ^10.1.2
```

Comment:
Changelog:
      https://pub.dev/packages/[file_picker/changelog#1012](https://github.com/file_picker/changelog/issues/1012)

This link seems to have gotten corrupted :slightly_smiling_face: